.env

```diff
@@ -0,0 +1 @@
+PGDATABASE = nc_news_test
```

Comment:
We should not post environment variables files to Github. Please add `.env.*`  to your gitignore file and remove this from your repo